app/bot.go

```diff
@@ -22,6 +23,29 @@ func (a *App) CreateBot(bot *model.Bot) (*model.Bot, *model.AppError) {
 		return nil, result.Err
 	}
 
+	// Get the owner of the bot, if one exists. If not, don't send a message
+	ownerUser, _ := a.Srv.Store.User().Get(bot.OwnerId)
```

Comment:
Can we check the error here and print an error log (if it's not just a "not found" error)? We try to avoid silently ignoring errors when possible

Reply:
Done! I wasn't too sure what to do with the error, so I'm logging and returning a nil, since I assume if there's an error outside of 'not found' we should be doing something about that.